packages/trader/src/RangeTrader.ts

```diff
@@ -151,7 +152,7 @@ export class RangeTrader {
 
     // Get the post trade spot price to double check deviation error.
     await this.tokenPriceFeed.update();
-    const exchangeSpotPriceAfterTrade = this.tokenPriceFeed.getCurrentPrice();
+    const exchangeSpotPriceAfterTrade = this.tokenPriceFeed.getLastBlockPrice();
```

Comment:
we should actually avoid the twap to show the exact spot price after the swap.

Reply:
right now the bot will create a default Uniswap feed for the synthetic provided. By doing it this way we don't need to do any configuration changes on the default feed created.

Reply:
But isn't it just as trivial to add an override config that contains a `twapLength: 0` parameter?

Reply:
this did not work as expected in unit tests. the feed would return no prices. in production, this is a bit irritating as it bloats the config more. Alternatively, we could get the `index` entry to inject this config but that seems more complex.

Do you see any downside with just using the `getLatestBlockTime`? this trader only works with uniswap right now (and therefore Uniswap feeds) so there will always be this method.